### ISSUE TITLE
get_metadata_constant return value not decoded

### ISSUE THREAD

**Issue by carumusan** [polkascan/py-substrate-interface#103]
```
substrate_connection.get_metadata_constant(
        module_name='Staking',
        constant_name='SessionsPerEra'
    )
```

Returns the value "0x060000" which has yet to be decoded from scale.

**Comment by arjanz** [comment]
As a workaround until fix is released:

```python
constant = substrate_connection.get_metadata_constant("Staking", "SessionsPerEra")
decoded_value = substrate_connection.decode_scale(constant.type, ScaleBytes(constant.constant_value))
```

**Comment by arjanz** [comment]
Released fix in [0.13.0](https://github.com/polkascan/py-substrate-interface/releases/tag/v0.13.0)

**Comment by arjanz** [comment]
since v0.13.4 there is a dedicated function to return the decoded value of runtime constants:

```python
constant = substrate.get_constant("Balances", "ExistentialDeposit")

print(constant.value) # 10000000000
```
